packages/nhost-js/src/utils/types.ts

```diff
@@ -73,6 +106,9 @@ export type FunctionCallResponse<T = unknown> =
     }
 
 export interface GraphqlResponse<T = object> {
-  errors?: object[]
+  errors?: GraphQLError[]
   data?: T
 }
+
+export type RestrictedFetchConfig = { headers?: Record<string, string> }
```

Comment:
Now that we are controlling the config, can't we just simply call it `NhostFetchConfig` or `NhostRequestConfig`?

Reply:
yep